### ISSUE TITLE
spaceSelectsMatch does not work (nor allowSpaces)

### ISSUE THREAD

**Issue by Pitouli** [zurb/tribute#450]
**How can we reproduce this bug?**

1. https://codepen.io/pitouli/pen/YzypxoP?editors=1010
2. Type '#B' -> 'Bob' highlights
3. Press space

**What did you expect to happen?**

'#B' should be replaced by a link with 'Bob Bill' inside.

**What happened instead?**

Nothing is replaced, and the list stays open, even if the parameter `allowSpaces` is at `false`

**Link**

https://codepen.io/pitouli/pen/YzypxoP?editors=1010

**Note** 

Pressing [enter] works as expected

**Comment by passcod** [comment]
Having hit that, I subsequently found that the option is "global", see #170.

**Comment by Pitouli** [comment]
I should convert that to feature request then! :) 

Because I do not want to allow spaces in hashtag (so space to confirm is good), but I do want to allow spaces in mentions (so I do not want space to confirm)

**Comment by passcod** [comment]
Yep, I have the exact same usecase... plus using the global flag with collections triggers the #495 error.

**Comment by brianarpie** [comment]
I have the exact same use case: I do not want to allow spaces in mentions, but I do want to allow spaces in hashtags.
